plugins/package-managers/maven/build.gradle.kts

```diff
@@ -45,6 +45,10 @@ dependencies {
     // container automatically. They are required on the classpath for Maven dependency resolution to work.
     runtimeOnly(libs.bundles.mavenResolver)
 
+    // Under certain circumstances, Tycho uses Wagon to download metadata for SNAPSHOT artifacts. Therefore, at
+    // least the wagon-http dependency should be available on the classpath.
+    runtimeOnly(libs.wagon.http)
```

Comment:
Hmm, I'm really surprised that this is needed: We already have all those Wagon / HTTP transports as part of `libs.bundles.mavenResolver`.